### DISCUSSION TITLE
Update Readme's indicators section with complete icon set

### DISCUSSION THREAD

**Discussion by danielnemenyi** [emacs-citar/citar#795] [Show and tell]
It took me a bit of time to set up `citar-indicators` nicely using all-the-icons. How about updating the [Indicators](https://github.com/emacs-citar/citar#indicators) section of the Readme to just give a complete copy-pastable setup for files, links and notes, and indicate how to customize it, rather than give a partial setup and let users figure out the rest?

![image](https://github.com/emacs-citar/citar/assets/8701448/88c143be-c8cc-4bd8-a5b4-771d9d553260)

Here's my code: 

``` elisp
(setq citar-indicators
  (list citar-indicator-files-icons
	citar-indicator-links-icons
        citar-indicator-notes-icons))

(defvar citar-indicator-files-icons
  (citar-indicator-create
   :symbol (all-the-icons-faicon
            "file-pdf-o"
            :face 'all-the-icons-red
	    :height 0.6
	    :v-adjust 0.1)
   :function #'citar-has-files
   :padding " "
   :tag "has:file"))

(defvar citar-indicator-links-icons
  (citar-indicator-create
   :symbol (all-the-icons-faicon
            "link"
            :face 'all-the-icons-yellow
	    :height 0.6
	    :v-adjust 0.1)
   :function #'citar-has-links
   :padding " "
   :tag "has:links"))

(defvar citar-indicator-notes-icons
  (citar-indicator-create
   :symbol (all-the-icons-faicon
            "file-text"
            :face 'all-the-icons-blue
	    :height 0.6
	    :v-adjust 0.1)
   :function #'citar-has-notes
   :padding " "
   :tag "has:notes"))

```

**Comment by bdarcus** [comment]
I did add that to the wiki; is it not linked from the README?

https://github.com/emacs-citar/citar/wiki/Indicators

EDIT: yes, it is linked there.

I don't want to burden the README with that because I want to encourage users to come up with different configs for this, and share them.

And I don't want to include them in the package itself because I don't want to add the dependency.

That reminds me: all-the-icons is in the process of switching to [SVG](https://github.com/domtronn/all-the-icons.el/tree/svg). I wonder how that might impact this?

**Reply by danielnemenyi** [reply]
> For other indicators, see the [wiki](https://github.com/emacs-citar/citar/wiki/Indicators).

Oh no I didn't see that! Sorry! I now see the text "For other indicators, see the [wiki](https://github.com/emacs-citar/citar/wiki/Indicators)."

> And I don't want to include them in the package itself because I don't want to add the dependency.

Of course.

Ok, a suggestion that might mean that overly hasty people like me grok that there's a wiki that we should turn to, would be to reduce the Indicators section text with...

> The UI includes configurable indicators which indicates the presence of different resources related to the reference:
> 
>     notes
>     library files
>     links
>     cited (for references cited in the current buffer)
> 
> By default plain text indicators are used, though they can be replaced with icons using all-the-icons.  

> ![image](https://github.com/emacs-citar/citar/assets/8701448/f76a434a-846d-4271-be4f-87ff8db4a678)

> See the [Wiki](https://github.com/emacs-citar/citar/wiki/Indicators) for more information.

Don't know what you think?

**Reply by bdarcus** [reply]
> ... a suggestion that might mean that overly hasty people like me grok that there's a wiki that we should turn to, would be to reduce the Indicators section text with...

I'm sort of agnostic, other than to say that the point of the indicators is one can put anything one wants there; plain text, unicode symbols, SVG icons, etc.

So any clarifying text wouldn't want to suggest otherwise, as the last clause you suggest would.

Feel free to submit a PR if you like ;-)
